src/work-allocation/containers/task-home/task-home.component.html

```diff
@@ -2,13 +2,27 @@
 
   <exui-info-message-container></exui-info-message-container>
 
-  <h3 class="govuk-heading-xl">{{pageTitle}}</h3>
-  <div class="govuk-grid-row">
-    <div class="govuk-grid-column-full">
-      <p>See your tasks and view tasks available for you to pick up.</p>
-      <xuilib-hmcts-sub-navigation [items]="subNavigationItems" label="Sub navigation"></xuilib-hmcts-sub-navigation>
-    </div>
-  </div>
+  <ng-container [ngSwitch]="featureVersion$ | async">
```

Comment:
Why is the switching logic in here in the template and not simply changing the already bound properties within the component? For example, the `pageTitle` property could simply be set to 'My work' and that would behave just fine and not need all this additional binding and switching.

Reply:
This is agreed as a feature toggle between different versions. I am ok with this logic